bucket/freetube.json

```
@@ -20,16 +20,17 @@
         ]
     ],
     "checkver": {
-        "url": "https://github.com/FreeTubeApp/FreeTube/tags",
-        "regex": "v([\\w.-]+)</a></h2>"
+        "url": "https://api.github.com/repos/FreeTubeApp/FreeTube/tags",
```

```suggestion
		"github": "https://github.com/FreeTubeApp/FreeTube"

```
This one will be simpler